python/packages/ai/teams/ai/citations/citations.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Optional
+
+
+@dataclass
```

Comment:
could we break up these dataclasses into their own respective separate files, thank you!

Reply:
is it okay if I don't? I did this similar to `message.py` is done, and there are multiple in one file. All the dataclasses are related to each other, as well.

Reply:
Spoke with Lily 1:1 -- if this is really bothersome I will fix it after Build.